elasticsearch/sgconfig/config.yml

```diff
@@ -4,8 +4,8 @@ opendistro_security:
       xff:
         enabled: true
         remoteIpHeader: 'x-forwarded-for'
+        internalProxies: '0:0:0:0:0:0:0:1|127\.0\.0\.1'
```

Comment:
shouldn't this be 'localhost'?
https://github.com/openshift/elasticsearch-proxy/commit/afa4328776afa3c253781217168a2c82482e7983

Reply:
This is compared against the socket's hostAddress

https://github.com/opendistro-for-elasticsearch/security/blob/703253ed58e4471d5573e9bf950637797c534b55/src/main/java/com/amazon/opendistroforelasticsearch/security/http/RemoteIpDetector.java#L132